content/blog/2022-02-25-react-v18-rc.md

```diff
@@ -0,0 +1,213 @@
+
+* `useID` is a new hook for generating unique IDs on both the client and server, while avoiding hydration mismatches. This solves an issue that already exists in React 17 and below, but it's even more important in React 18 because of how our streaming server renderer delivers HTML out-of-order. For more information see the post [here](https://github.com/reactwg/react-18/discussions/111).
+* `useSyncExternalStore` is a new hook that allows external stores to support concurrent reads by forcing updates to the store to be synchronous. This new API is recommended for any library that integrates with state external to React. For more information, see the overview [here](https://github.com/reactwg/react-18/discussions/70) and API details [here](https://github.com/reactwg/react-18/discussions/86).
+* `useInsertionEffect` is a new hook that allows CSS-in-JS libraries to address performance issues of injecting styles in render. This hook will run after the DOM is mutated, but before layout effects read the new layout. This solves an issue that already exists in React 17 and below, but is even more important in React 18 because React yields to the browser during concurrent rendering, giving it a chance to recalculate layout. For more information, see the post [here](https://github.com/reactwg/react-18/discussions/110).
+
+React 18 also introduces new APIs for concurrent rendering such as `startTransition` and `useDeferredValue`, which we will share more about in the upcoming stable release post.
+
+## Adding Strict Effects to Strict Mode
+
+In the future, we'd like to add features to React which would allow a component to mount without immediately creating effects (such as pre-rendering), or to destroy effects in already-mounted components (such as when a component becomes no longer visible). These features will add better performance and resource management out-of-the-box to React, but require effects to be decoupled from the component lifecycle and resilient to being created and destroyed multiple times in a component.
+
+To help surface these issues, React 18 introduced Strict Effects to Strict Mode. With Strict Effects, React will automatically destroy and re-create every effect in development, whenever a component mounts.
+
+For more information, see the Working Group posts for [Adding Strict Effects to Strict Mode](https://github.com/reactwg/react-18/discussions/19) and [How to Support Strict Effects](https://github.com/reactwg/react-18/discussions/18).
+
+## Testing
+
+TODO:
+* `IS_REACT_ACT_ENVIRONMENT` https://github.com/reactwg/react-18/discussions/102
+
+## Dropping support for Internet Explorer
+
+In this release, React dropping support for Internet Explorer, which is going end of life on June 15, 2022. We’re making this change now because new features introduced in React 18 are built using modern browser features such as microtasks which cannot be adequately polyfilled in IE.
```

Comment:
```suggestion
In this release, React is dropping support for Internet Explorer, which is going end of life on June 15, 2022. We’re making this change now because new features introduced in React 18 are built using modern browser features such as microtasks which cannot be adequately polyfilled in IE.
```